Project-Lily/Project-Lily/DogListViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DogListViewModel.swift
+//  Project-Lily
+//
+//  Created by Kendall Poindexter on 11/17/18.
+//  Copyright © 2018 Kendall Poindexter. All rights reserved.
+//
+
+import Foundation
+
+//class ViewModel {
+//    var dogs = [Dog]()
+//
+//    func getDogs() {
+//        let gotDogsCompletion: ([Dog]) -> Void = { newDogs in
+//                self.dogs = newDogs
+//                // delegate?.didFetchDogs()
+//        }
+//
+//        NetworkManager().createDogURLSession(completion: gotDogsCompletion)
+//    }
+//}
```

Comment:
1) This looks promising!
2) Is this just to collect some notes as you're working to build out a ViewModel?

Reply:
Yes! That is exactly what that code is. I am going to work with it more next, but I wanted to modify it in a separate PR